### PR TITLE
add debugFillDescription to OverflowBox

### LINKED ISSUE #1487: OverflowBox lacks a debugFillDescription

### PR DESCRIPTION
**Pull request by yjbanov**
Fixes #1487

### CODE REVIEW

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -655,6 +655,19 @@ class OverflowBox extends OneChildRenderObjectWidget {
       ..maxHeight = maxHeight
       ..alignment = alignment;
   }
+
+  @override
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: we don't yet use @override.

File: packages/flutter/test/widget/overflow_box_test.dart [hunk 2]
```diff
@@ -32,4 +32,20 @@ void main() {
       expect(box.size, equals(const Size(100.0, 50.0)));
     });
   });
+
+  test('OverflowBox implements debugFillDescription', () {
+    var description = [];
```
Comment (Hixie) [inline packages/flutter/test/widget/overflow_box_test.dart]: no 'var's, always type everything.
also, type the literal:  `<String>[]`

**Comment by Hixie** [comment]
lgtm

File: packages/flutter/lib/src/widgets/basic.dart (hunk 1 above)
Comment (sethladd) [inline packages/flutter/lib/src/widgets/basic.dart]: My understanding is that we can use `@override` when https://github.com/flutter/flutter/issues/1220 is possible.

File: packages/flutter/test/widget/overflow_box_test.dart (hunk 2 above)
Comment (sethladd) [inline packages/flutter/test/widget/overflow_box_test.dart]: And, when https://github.com/flutter/flutter/issues/1220 lands, we'll be able to write a rule for this, too :)

cc @pq

File: packages/flutter/lib/src/widgets/basic.dart (hunk 1 above)
Comment (yjbanov) [inline packages/flutter/lib/src/widgets/basic.dart]: done

File: packages/flutter/test/widget/overflow_box_test.dart (hunk 2 above)
Comment (yjbanov) [inline packages/flutter/test/widget/overflow_box_test.dart]: done.
done.

**Pull request merged**